### ISSUE TITLE
Test using pytest?

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/kastore#139]
At the moment the dev requirements for kastore list `nose`. Are we switching to pytest?

**Comment by jeromekelleher** [comment]
Sure - there's a bit of work involved though, and it's not a high priority.

**Comment by hyanwong** [comment]
Yep - leaving this open so we don't forget, though. Same with tsinfer, I guess.

**Comment by jeromekelleher** [comment]
Note we started running tests using pytest in #163

**Comment by hyanwong** [comment]
Shall I close this then?

**Comment by benjeffery** [comment]
I changed as nose doesn't work with 3.10, which kastore now supports.
